src/doc_search/workflow/__init__.py

```diff
@@ -82,11 +84,29 @@ def execute(self) -> dict:
         }
 
 
+class ImageMagickCommand(WorkflowBase):
+    """
+    Use command based on OS
+    """
+
+    def execute(self) -> dict:
+        command: Optional[str] = "convert"
+
+        if platform.system() == "Windows":
+            command = os.getenv("IMCONV")
+            assert command is not None, (
+                "IMCONV environment variable not set. It should point to location of " "ImageMagick's convert.exe"
```

Comment:
"IMCONV environment variable not set. It should point to location of " "ImageMagick's magick.exe"